.devcontainer/devcontainer.json

```diff
@@ -70,5 +70,8 @@
     "containerEnv": {
         "PIPENV_VERBOSITY": "-1",
         "DEVELOPMENT_MODE": "1"
-    }
+    },
+    "mounts": [
+        "source=${localEnv:HOME}/.ssh,target=/home/vscode/.ssh,type=bind,consistency=cached"
+    ]
```

Comment:
That is indeed a security risk. Solve this by using ssh-agent, which works in the container instead. Not sure if that's a thing in macOS, but I believe it is. Using that the container talks to a local port and never exposes your private keys to what can be arbitrary codes. With this in place, I cannot even try this code out.